pkg/api/scylla/v1/types_cluster.go

```diff
@@ -524,16 +524,66 @@ type Storage struct {
 	StorageClassName *string `json:"storageClassName,omitempty"`
 }
 
+type TLSCertificateType string
+
+const (
+	TLSCertificateTypeOperatorManaged TLSCertificateType = "OperatorManaged"
+	TLSCertificateTypeUserManaged     TLSCertificateType = "UserManaged"
+)
+
+type UserManagedTLSCertificateOptions struct {
+	// secretName references a kubernetes.io/tls type secret containing the TLS cert and key.
+	SecretName string `json:"secretName"`
+}
+
+type OperatorManagedTLSCertificateOptions struct {
+	// additionalDNSNames represents external DNS names that the certificates should be signed for.
+	// +optional
+	AdditionalDNSNames []string `json:"additionalDNSNames,omitempty"`
```

Comment:
Is it me not being able to find it, or is it not being used/tested anywhere?

Reply:
nope, that's missing, thx

Reply:
done